### PR TITLE
MVPoly: add tests checking add and mul by one and zero

### PR DESCRIPTION
**Pull request by dannywillems**

### CODE REVIEW

File: mvpoly/tests/prime.rs [hunk 1]
```diff
@@ -151,3 +151,36 @@ fn test_mul() {
...
 }
+
+#[test]
+fn test_mul_by_one() {
+    let mut rng = o1_utils::tests::make_test_rng(None);
+    let p1 = Dense::<Fp, 7, 2>::random(&mut rng);
+    let one = Dense::<Fp, 7, 2>::one();
+    let p2 = p1.clone() * one.clone();
+    assert_eq!(p1.clone(), p2);
+    let p3 = one * p1.clone();
```
[the remark this answers is not in the document]
Comment (dannywillems) [reply mvpoly/tests/prime.rs:162]: It is not as it won't be used anymore after.

**Review by Fizzixnerd — APPROVED** [review]
